client/components/theme-tier/theme-tier-badge/theme-tier-bundled-badge.js

```diff
@@ -0,0 +1,53 @@
+import { BundledBadge, PremiumBadge } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import useBundleSettings from 'calypso/my-sites/theme/hooks/use-bundle-settings';
+import { useSelector } from 'calypso/state';
+import { canUseTheme } from 'calypso/state/themes/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import ThemeTierBadgeTracker from './theme-tier-badge-tracker';
+
+export default function ThemeTierBundledBadge( { themeId } ) {
+	const translate = useTranslate();
+	const siteId = useSelector( getSelectedSiteId );
+	const bundleSettings = useBundleSettings( themeId );
+	const legacyCanUseTheme = useSelector(
+		( state ) => siteId && canUseTheme( state, siteId, themeId )
+	);
+
+	if ( ! bundleSettings ) {
+		return;
+	}
+
+	const BadgeIcon = bundleSettings.iconComponent;
+
+	return (
```

Comment:
Hi @Copons!

Not sure if it was intentional, but I noticed that these changes impacted the behavior of the bundled badges.
In the WooCommerce themes, like "Tsubaki", it's displaying 2 badges and not displaying the tooltip in the WooCommerce badge.

<img width="384" alt="Screenshot 2023-12-11 at 18 58 18" src="https://github.com/Automattic/wp-calypso/assets/876340/379cc829-e29a-4b02-b5a6-ac0e95271c30">

Reply:
@renatho Yes, that's the expected behaviour, **as long as it's behind feature flag**! 😅 

The UI of the new Tier system is not final, though, and won't for a few more weeks at least. You can chime in on P2: p9Jlb4-a7w-p2#comment-10152 🙂

Reply:
Ah, perfect! =)
Thank you for the answer and for the reference, @Copons!